pymongo_test_query.py

```diff
@@ -1,38 +1,43 @@
 # Get the database using the method we defined in pymongo_test_insert file 
 from pymongo_test_insert import get_database
-dbname = get_database()
+# dbname = get_database()
 
-# Create a new collection
-collection_name = dbname["user_1_items"]
+def query():
+	dbname = get_database()
+	# Create a new collection
+	collection_name = dbname["user_1_items"]
 
-item_details = collection_name.find()
-for item in item_details:
-	# This will give readable output, but KeyError
-	print(item['item_name'], item['category'])
+	item_details = collection_name.find()
+	# for item in item_details:
+	# 	# This will give readable output, but KeyError
+	# 	print(item['item_name'], item['category'])
 
-###---------------------------------------------------###
-### Comment the above 'for loop' & 'print statements' ### 
-###       for the next lines of code to work          ###
-###---------------------------------------------------###
-from pandas import DataFrame
+	###---------------------------------------------------###
+	### Comment the above 'for loop' & 'print statements' ### 
+	###       for the next lines of code to work          ###
+	###---------------------------------------------------###
+	from pandas import DataFrame

```

Comment:
Let's cut all that and simplify it to levels of other examples

Reply:
I am only sending a `ping` command now. I am not sure if that's too simplified.